Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -249,15 +283,20 @@ void ScrollAnchoringController::updateAnchorElement()
 
 void ScrollAnchoringController::adjustScrollPositionForAnchoring()
 {
-    SetForScope midUpdatingScrollPositionForAnchorElement(m_midUpdatingScrollPositionForAnchorElement, true);
     auto queued = std::exchange(m_isQueuedForScrollPositionUpdate, false);
+    auto supressed = std::exchange(m_shouldSupressScrollPositionUpdate, false);
     if (!m_anchorElement || !queued)
         return;
     auto renderBox = m_anchorElement->renderer();
```

Comment:
`renderer()` returns a `RenderObject`, not a `RenderBox`, so calling this `renderBox` is misleading.

Reply:
Fixed.